src/webgpu/api/operation/vertex_state/correctness.spec.ts

```diff
@@ -405,6 +423,28 @@ struct VSOutputs {
       }
 
       case 'unorm': {
+        if (formatInfo.bytesPerComponent === 'packed') {
+          assert(format === 'rgb10a2'); // This is the only packed format for now.
+          assert(bitSize === 0);
+
+          /* prettier-ignore */
+          const data = [
+            [0, 0, 0, 0],
+            [1023, 1023, 1023, 3],
+            [243, 567, 765, 2],
+          ];
+          const vertexData = new Uint16Array(data.map(makeRgb10a2)).buffer;
```

Comment:
Why `Uint16Array`? There will be a data conversion happening that's not what we want. Uint32Array would be better. Also why is expectedData not a Float32Array?

Reply:
I now use `Uint32Array` and `Float32Array` respectively for `vertexData` and `expectedData`.